packages/web-components/fast-components/src/accordion-item/accordion-item.styles.ts

```diff
@@ -58,17 +52,17 @@ export const accordionItemStyles = (context, definition) =>
         padding: 0 calc((6 + (${designUnit} * 2 * ${density})) * 1px);
         text-align: left;
         height: calc(${heightNumber} * 1px);
-        color: ${neutralForegroundRestBehavior.var};
+        color: ${neutralForegroundRest};
         cursor: pointer;
         font-family: inherit;
     }
 
     .button:hover {
-        color: ${neutralForegroundHoverBehavior.var};
+        color: ${neutralForegroundRest};
     }
 
     .button:active {
-        color: ${neutralForegroundActiveBehavior.var};
+        color: ${neutralForegroundRest};
     }
```

Comment:
@nicholasrice should these all be neutralForegroundRest or should we also have Hover and Active here?

Reply:
In practice `neutralForeground` never actually had hover/active states. They were all the same, so I just made a single neutralForegroundRest recipe.

Reply:
makes sense!